apps/backend/src/libs/helpers/transform-to-snake-case/transform-to-snake-case.helper.ts

```diff
@@ -0,0 +1,7 @@
+import snakeCase from "lodash/snakeCase.js";
+
+const transformToSnakeCase = (name: string): string => {
+	return snakeCase(name);
```

Comment:
I don't think it's a good idea to install lodash for this. You can create your own snakeCase

![image](https://github.com/user-attachments/assets/4b00946f-539e-4bc3-b62c-ae2426c77025)